### PR TITLE
Review comments in code

### PR DESCRIPTION
**Pull request by Lukan707**

### CODE REVIEW

**Comment by Lukan707** [comment]
By mistake, I apperantly created the branch out of the branch from pull request https://github.com/ITU-BDSA23-GROUP4/Chirp/pull/290, and not main.

So, this pull request is dependent on pull request https://github.com/ITU-BDSA23-GROUP4/Chirp/pull/290 being merged into main first.

PLEASE NOTICE: DEPENDENT ON PULL REQUEST https://github.com/ITU-BDSA23-GROUP4/Chirp/pull/290

**Review by SkarpKat — APPROVED** [review]
Very nice

File: src/Chirp.Infrastructure/AuthorRepository.cs [hunk 1]
```diff
@@ -209,18 +194,14 @@ public async Task AddFollowee(string _AuthorName, string _FolloweeName)
             }
         }
 
-        private static Task<List<CheepDTO>> GetAllCheepsFromAuthorAsync(string Name, ChirpDBContext _dbcontext) {
-            return Task.Run(() => GetAllCheepsFromAuthor(Name, _dbcontext));
-        }
-
+        // Helper method: Retrieves all cheeps related to the given author from the db.
```
Comment (Faberen) [inline src/Chirp.Infrastructure/AuthorRepository.cs:197]: Maybe redundant information?

File: src/Chirp.Infrastructure/AuthorRepository.cs [hunk 2]
```diff
@@ -237,10 +218,7 @@ private  static List<CheepDTO> GetAllCheepsFromAuthor(string Name, ChirpDBContex
             }
         }
 
-        private static Task<List<AuthorDTO>> GetAllFollowedAuthorAsync(Guid AuthorId, ChirpDBContext _dbcontext) {
-            return Task.Run(() => GetAllFollowedAuthor(AuthorId, _dbcontext));
-        }
-
+        // Helper method: Retrieves all followed authors from the AuthorAuthor entity in the db.
```
Comment (Faberen) [inline src/Chirp.Infrastructure/AuthorRepository.cs:221]: Maybe redundant information?

File: src/Chirp.Infrastructure/AuthorRepository.cs [hunk 3]
```diff
@@ -266,10 +244,8 @@ private static List<AuthorDTO> GetAllFollowedAuthor(Guid AuthorId, ChirpDBContex
 
             return followed;
         }
-
-        private static Task<List<AuthorDTO>> GetAllFollowersAsync(Guid AuthorId, ChirpDBContext _dbcontext) {
-            return Task.Run(() => GetAllFollowers(AuthorId, _dbcontext));
-        }
+        
+        // Helper method: Retrieves all followers (authors) from the AuthorAuthor entity in the db.
```
Comment (Faberen) [inline src/Chirp.Infrastructure/AuthorRepository.cs:248]: Maybe redundant information?

**Review by Faberen — APPROVED** [review]

**Review by AlbinoLoaf — APPROVED** [review]

**Pull request merged**